apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -416,46 +421,84 @@ NetSimPacketEditor.prototype.makeBlurHandler = function (fieldName, converterFun
  */
 
 /**
+ * Convert binary to an integer, intentionally limiting the binary width so
+ * that overflow can occur.
+ * @param {string} binaryString (interpreted as unsigned)
+ * @param {number} maxWidth in bits
+ * @returns {number}
+ */
+var truncatedBinaryToInt = function (binaryString, maxWidth) {
+  return binaryToInt(binaryString.substr(-maxWidth));
+};
```

Comment:
I tend to prefer function foo() vs. var foo = function() in most cases, though I don't see anything about this in our style guide.  The one exception (for me) is when I'm creating a function within another function. I'm not sure I have any good justification for this, and would be interested to hear if you have a preference/why
```
function foo () {
  var bar = function () {} 
}